chroma-agent/chroma_agent/action_plugins/detect_scan.py

```diff
@@ -15,8 +15,10 @@
 
 class LocalTargets(object):
     '''
-    Allows local targets to be examined. Not the targets are only examined once with the results cached. Detecting change
-    therefore requires a new instance to be created and queried.
+    Allows local targets to be examined.
+    Note the targets are only examined once with the results cached.
+    Detecting change therefore requires a new instance
+    to be created and queried.
     '''
 
     def __init__(self, target_devices):
```

Comment:
We should have all the info we need in device-scanner / aggregator now.

Reply:
will be replacing/removing both detect_scan and lustre in future commits (still to do)